apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -0,0 +1,696 @@
+                    />
+                  </FormControl_Shadcn_>
+                </FormItemLayout>
+              )}
+            />
+
+            {['Problem', 'Database_unresponsive', 'Performance'].includes(category) && (
+              <FormField_Shadcn_
+                name="allowSupportAccess"
+                control={form.control}
+                render={({ field }) => (
+                  <FormItemLayout
+                    layout="flex"
+                    className="px-6"
+                    label="Allow Supabase Support to access your project temporarily"
+                    description="In some cases, we may require temporary access to your project to complete troubleshooting, or to answer questions related specifically to your project"
+                  >
+                    <FormControl_Shadcn_>
+                      <Checkbox_Shadcn_
+                        {...field}
+                        value={String(field.value)}
+                        checked={field.value}
+                        onCheckedChange={(value) => field.onChange(value)}
+                      />
+                    </FormControl_Shadcn_>
+                  </FormItemLayout>
+                )}
+              />
+            )}
+
+            <div className="px-6">
+              <div className="flex flex-col gap-y-1">
+                <p className="text-sm text-foreground-light">Attachments</p>
+                <p className="text-sm text-foreground-lighter">
+                  Upload up to {MAX_ATTACHMENTS} screenshots that might be relevant to the issue
+                  that you're facing
+                </p>
+              </div>
+              <input
+                multiple
+                type="file"
+                ref={uploadButtonRef}
+                className="hidden"
+                accept="image/png, image/jpeg"
+                onChange={onFilesUpload}
+              />
+              <div className="flex items-center gap-x-2 mt-4">
+                {uploadedDataUrls.map((x: any, idx: number) => (
+                  <div
+                    key={idx}
+                    style={{ backgroundImage: `url("${x}")` }}
+                    className="relative h-14 w-14 rounded bg-cover bg-center bg-no-repeat"
+                  >
+                    <div
+                      className={[
+                        'flex h-4 w-4 items-center justify-center rounded-full bg-red-900',
+                        'absolute -top-1 -right-1 cursor-pointer',
+                      ].join(' ')}
+                      onClick={() => removeUploadedFile(idx)}
+                    >
+                      <X size={12} strokeWidth={2} />
+                    </div>
+                  </div>
+                ))}
+                {uploadedFiles.length < MAX_ATTACHMENTS && (
+                  <div
+                    className={[
+                      'border border-stronger opacity-50 transition hover:opacity-100',
```

Comment:
you could import and use ButtonVariants({type: 'default'}) and then `hover:border-stronger` so it looks like a normal button, but you've applied the styles to a `<div />`

Reply:
you have a example RE ButtonVariants? i cant seem to find any usage in dashboard or ui package atm

but i'll leave this out of this PR too - since this PR focuses on the refactoring of the react hook form parts